lib/usecases/store_auth_token_use_case.dart

```diff
@@ -17,6 +20,7 @@ class StoreAuthTokenUseCase extends UseCase<void, LoginModel> {
       await _secureStorage.writeSecureData(accessTokenKey, input.accessToken);
       await _secureStorage.writeSecureData(refreshTokenKey, input.refreshToken);
       await _secureStorage.writeSecureData(tokenTypeKey, input.tokenType);
+      _sharedPreferencesStorage.saveLoginStatus(true);
```

Comment:
@Shayokh144 @Tuubz Why don't we rely on the existence of `accessToken` (and maybe `tokenType`) to verify this status? In this way, we can get rid of adding the `shared_preferences` dependency?

Reply:
@luongvo I added `shared_preferences` so that I can get the login status immediately. fro secure_storage reading a value is an async operation which needs some time, the way I am navigation I need the login status immediately otherwise I need to use [Future Builder](https://api.flutter.dev/flutter/widgets/FutureBuilder-class.html) which requires a new View that can be shown when we are waiting for the login status. As I don't want to add a new screen  I used `shared_preferences` to read the login status immediately and load HomeScreen() or LoginScreen().